src/tasks/tasks.service.ts

```diff
@@ -25,4 +24,11 @@ export class TasksService {
   async updateTask(id: number, updateTaskDto: UpdateTaskDto): Promise<Task> {
     return await this.taskRepository.updateTask(id, updateTaskDto);
   }
+
+  async deleteTask(id: number) {
```

Comment:
```suggestion
  async deleteOne(id: number) {
```

No need to mention task explicitly in the name as you'd call this method like so: `this.tasksService.deleteOne` in other files.